docs/en/creating-widgets/supplemental.md

```diff
@@ -1,4 +1,4 @@
-# Introduction to widgets
+# Creating widgets
```

Comment:
Same title as the actual guide itself? Do you think the overall guide name should be changed?

Reply:
Think the guide name makes sense - I've changed this to something else now that isn't just duplicating the guide name.